Inspicio/wwwroot/js/multipleImages.js

```diff
@@ -10,30 +10,35 @@
         var description = $("#Image_Description_UserInput").val();
         var content = $("#b64uploader").val();
         if (title.length !== 0 && content.length !== 0) {
-
-            $("#uploader").css("width", "100%");
-            $(this).removeClass("ready");
-            var contentCol = listOfImages.map(v => v.Content);
-            if ($.inArray(content, contentCol) > -1) {
-             alert("This image is already added");
-             $("#uploader").html("").append(dropzoneMainText);
-             $(dropzoneMainText).show();
+            if (title.length < 5) {
+                alert("Screen title must have at least 5 characters!")
```

Comment:
Ah, OK. I'm still not sure why the title needs to be at least 5 chars (I do get that you need to check the rule here because somewhere else assumes it's at least 5 chars).